### ISSUE TITLE
0.14 Breeze theme does not honor background image setting

### ISSUE THREAD

**Issue by geckolinux** [sddm/sddm#772]
Hi, it looks like the new SDDM 0.14 Breeze theme does not honor the specified background image setting. It just shows a sky blue background.

Here's the `/etc/sddm.conf`
```
[Autologin]
Relogin=false
User=

[General]
HaltCommand=
RebootCommand=

[Theme]
Current=breeze
CursorTheme=breeze_cursors

[Users]
MaximumUid=65000
MinimumUid=1000

[XDisplay]
DisplayCommand=/etc/X11/xdm/Xsetup
MinimumVT=7
```

And here's `/usr/share/sddm/themes/breeze/theme.conf.user`
```
[General]
background=studio_wallpaper.jpg
```

I confirmed that `/usr/share/sddm/themes/breeze/studio_wallpaper.jpg` is present and readable.

Thanks for taking a look!

**Comment by Dav1dde** [comment]
I can confirm this, replacing `theme.conf` with `theme.conf.user`:

```
[General]
background=my_wallpaper.jpg
```

just yields a white-ish background, most likely the default color.

**Comment by arunsai123** [comment]
I can confirm this. I can't change background image in breeze theme. I use sddm-kcm to change backgrounds

**Comment by geckolinux** [comment]
Appears to be fixed in 5.9.1:
https://bugs.kde.org/show_bug.cgi?id=370521
